ddtrace/contrib/grpc/patch.py

```diff
@@ -30,15 +30,25 @@ def unpatch():
 
 def _insecure_channel_with_interceptor(wrapped, instance, args, kwargs):
     channel = wrapped(*args, **kwargs)
-    target = args[0]
+
+    if kwargs.get('target'):
```

Comment:
We might want to do `if 'target' in kwargs:` instead. In case `kwargs['target']` is falsey.

Reply:
Good call, I'll make that change.